File: data/museum/oseg_generated/ruby/GetSpecialEvent_default_example.rb
require "openapimuseum_client"

OpenapiMuseumClient.configure do |config|
end

begin
    api_caller = OpenapiMuseumClient::EventsApi.new

    response = api_caller.get_special_event(
        "dad4bce8-f5cb-4078-a211-995864315e39",
    )

    p response
rescue OpenapiMuseumClient::ApiError => e
    puts "Exception when calling Events#get_special_event: #{e}"
end

File: data/museum/oseg_generated/php/GetSpecialEvent_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPIMuseum\Client\Configuration::getDefaultConfiguration();

try {
    $api_caller = new OpenAPIMuseum\Client\Api\EventsApi(config: $config);

    $response = $api_caller->getSpecialEvent(
        event_id: "dad4bce8-f5cb-4078-a211-995864315e39",
    );

    print_r($response);
} catch (OpenAPIMuseum\Client\ApiException $e) {
    echo 'Exception when calling Events#getSpecialEvent: '
        . print_r($e->getResponseObject());
}
